Assignment4/JustAntoRS-z170035/Task06.ipynb

```diff
@@ -0,0 +1,205 @@
+        "g.add((ns.Researcher, RDF.type, RDFS.Class))\n",
+        "for s, p, o in g:\n",
+        "  print(s,p,o)"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qp1oe2Eddsvo",
+        "colab_type": "text"
+      },
+      "source": [
+        "**TASK 6.1: Create a new class named \"University\"**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "tags": []
+      },
+      "outputs": [],
+      "source": [
+        "g.add((ns.University, RDF.type, RDFS.Class))\n",
+        "for s,p,o in g:\n",
+        "    print(s,p,o)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MXBqtBkJd22I",
+        "colab_type": "text"
+      },
+      "source": [
+        "**TASK 6.2: Add \"Researcher\" as a subclass of \"Person\"**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "g.add((ns.Researcher, RDFS.subClassOf, ns.Person))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "OGct6k7Ld9O0",
+        "colab_type": "text"
+      },
+      "source": [
+        "**TASK 6.3: Create a new individual of Researcher named \"Jane Smith\"**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "g.add((ns.JaneSmith, RDF.type, ns.Researcher))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "tD383J__eHfV",
+        "colab_type": "text"
+      },
+      "source": [
+        "**TASK 6.4: Add to the individual JaneSmith the fullName, given and family names**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "vcard = Namespace(\"http://www.w3.org/2001/vcard-rdf/3.0#\") \n",
+        "# Fullname\n",
+        "g.add((ns.JameSmith, vcard.FN, Literal(\"Jane Smith\")))\n",
+        "# Given name\n",
+        "g.add((ns.JameSmith, vcard.Given, Literal(\"Jane\")))\n",
+        "# Family Name\n",
+        "g.add((ns.JameSmith, vcard.Family, Literal(\"Smith\")))"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "GyZOMndoeUj4",
+        "colab_type": "text"
+      },
+      "source": [
+        "**TASK 6.5: Add UPM as the university where John Smith works**"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Add the University to the graph\n",
+        "g.add((ns.UPM, RDF.type, ns.University))\n",
+        "# John Smith works in UPM\n",
+        "g.add((ns.JohnSmith, vcard.Work, ns.UPM))"
```

Comment:
this property does not exist in vcard vocabulary, and a colleague from you made the same mistake ;-)

Reply:
Yes, I tried searching for a property for working in the web "http://www.w3.org/2001/vcard-rdf/3.0#" and I failed at it :sweat_smile: so I took a look at what property other students have used here.

Thanks for spotting this out! :)